aptos-move/framework/aptos-framework/sources/account.spec.move

```diff
@@ -575,6 +575,7 @@ spec aptos_framework::account {
         // This function should not abort assuming the result of `sha3_256` is deserializable into an address.
         aborts_if [abstract] false;
         ensures [abstract] result == spec_create_resource_address(source, seed);
+        ensures [abstract] source != result; // We can assume that the derived resource account does not equal to `source`
```

Comment:
Could we assume this is correct?

Reply:
Yes, I think we can assume that they are practically different because `result = hash (source ++ seed ++ salt)`